libs/clients/national-registry/v3/src/lib/nationalRegistryV3.config.ts

```diff
@@ -0,0 +1,38 @@
+import { defineConfig } from '@island.is/nest/config'
+import { z } from 'zod'
+
+const schema = z.object({
+  fetchTimeout: z.number().int(),
+  clientId: z.string(),
+  clientSecret: z.string(),
+  scope: z.string(),
+  endpoint: z.string(),
+  xRoadServicePath: z.string(),
+})
+
+export const NationalRegistryV3ClientConfig = defineConfig<
+  z.infer<typeof schema>
+>({
+  name: ' NationalRegistryV3Client',
+  schema,
+  load: (env) => ({
+    fetchTimeout: 20000,
```

Comment:
20sec is a really long time, I would recommend to lower this to 10sec.

Reply:
isn´t there a default timeout anyway?